bindings/pyfiction/include/pyfiction/algorithms/simulation/sidb/quicksim.hpp

```diff
@@ -44,9 +44,7 @@ inline void quicksim(pybind11::module& m)
                        DOC(fiction_quicksim_params_iteration_steps))
         .def_readwrite("alpha", &fiction::quicksim_params::alpha, DOC(fiction_quicksim_params_alpha))
         .def_readwrite("number_threads", &fiction::quicksim_params::number_threads,
-                       DOC(fiction_quicksim_params_number_threads))
-
-        ;
+                       DOC(fiction_quicksim_params_number_threads));
```

Comment:
Please revert this change
```suggestion
                       DOC(fiction_quicksim_params_number_threads))
                       
                       ;
```